src/cli-validator/index.js

```diff
@@ -43,6 +43,10 @@ program
   .option(
     '-e, --errors_only',
     'only print the errors, ignore the warnings'
+  )
+  .option(
+    '--debug',
+    'enable debugging output'
```

Comment:
Will you document this new flag in the README?

Reply:
Fixed!